src/elements/content-sidebar/activity-feed/annotations/__tests__/AnnotationActivity.test.js

```diff
@@ -49,22 +48,7 @@ describe('elements/content-sidebar/ActivityFeed/annotations/AnnotationActivity',
         'data-resin-target': 'annotationLink',
     });
 
-    const defaultFeatures = {
-        activityFeed: {
-            modifyAnnotations: {
-                enabled: true,
-            },
-        },
-    };
-
-    const getWrapper = (props = {}) => shallow(<AnnotationActivity {...mockActivity} {...props} />);
-
-    const getWrapperWithFeatures = props =>
-        mount(
-            <FeatureProvider features={defaultFeatures}>
-                <AnnotationActivity {...mockActivity} {...props} />
-            </FeatureProvider>,
-        );
+    const getWrapper = (props = {}) => mount(<AnnotationActivity {...mockActivity} {...props} />);
```

Comment:
We need mount instead of shallow because we need to render `AnnotationActivityMenu`.

Reply:
Could do a shallow mount and `dive()` into the wrapper to access the `AnnotationActivityMenu` if that's what you want